### PR TITLE
Add filtering functionality to lunch menu

### PR DESCRIPTION
**Pull request by abjugard**
Especially useful for the CLI menu, allows creating aliases for the
lunch menu at a specific location or restaurant.

### CODE REVIEW

File: app/models/lunch.rb [hunk 1]
```diff
@@ -24,35 +24,35 @@ def einstein
...
-        #"Restaurang Hyllan" => "http://intern.chalmerskonferens.se/view/restaurant/hyllan/RSS%20Feed.rss?today=true&locale=#{locale}",
-        "L's Resto"         => "http://intern.chalmerskonferens.se/view/restaurant/l-s-resto/RSS%20Feed.rss?today=true&locale=#{locale}",
-        "Kokboken"          => "http://intern.chalmerskonferens.se/view/restaurant/kokboken/RSS%20Feed.rss?today=true&locale=#{locale}"}
+      restaurants = [
+        {name: "Linsen", url: "http://intern.chalmerskonferens.se/view/restaurant/linsen/RSS%20Feed.rss?today=true&locale=#{locale}", location: "Johanneberg"},
+        {name: "Kårrestaurangen", url: "http://intern.chalmerskonferens.se/view/restaurant/karrestaurangen/Veckomeny.rss?today=true&locale=#{locale}", location: "Johanneberg"},
+        {name: "L's kitchen", url: "http://intern.chalmerskonferens.se/view/restaurant/l-s-kitchen/Projektor.rss?today=true&locale=#{locale}", location: "Lindholmen"},
+        {name: "Express", url: "http://intern.chalmerskonferens.se/view/restaurant/express/V%C3%A4nster.rss?today=true&locale=#{locale}"},
```
Comment (malmers) [inline app/models/lunch.rb]: The location for Express is Johanneberg. Not to be confused with "Dagens express" at Lindholmen.

**Comment by malmers** [comment]
kummel

"Kummel" is missing from fish list, making it hard for rails to categorize the meal. Maybe there should be a fallback name when a meal can't be categorized? Or is it better to just assume that the first meal is fish and the second meat?

**Comment by abjugard** [comment]
Assumptions are bad I think, better to have a fallback "food" or something, but adding "kummel" might be a good idea.

**Comment by malmers** [comment]
Maybe it's a good idea to add the fish "sej" as well.

Today's menu had problem with "oxkub":
oxkub

The menu seems to be consistent with the food ordering. I prefer ordering to matching. We should at least use ordering as a fallback when matching for fish or meat.

**Pull request merged**